localstack/services/s3/provider.py

```diff
@@ -556,6 +561,60 @@ def complete_multipart_upload(
         self._notify(context)
         return response
 
+    @handler("ListMultipartUploads", expand=False)
+    def list_multipart_uploads(
+        self,
+        context: RequestContext,
+        request: ListMultipartUploadsRequest,
+    ) -> ListMultipartUploadsResult:
+
+        try:
+            response: ListMultipartUploadsResult = call_moto(context)
+        except NotImplementedError:
```

Comment:
just a question: do we need this try/except? is this method implemented in moto, and are we only using it partially? or can we just replace `call_moto` with our implementation all together?

Reply:
It's partially implemented in moto, and raises only if some parameters are passed:
https://github.com/localstack/moto/blob/localstack/moto/s3/responses.py#L428-L432

We could replace it I suppose because I think if there's an exception, we're basically doing what it would do if there wasn't an exception. Good idea!

Reply:
I guess it was in the case they would add more things, because right now our implementation is incomplete, and if moto adds feature we wouldn't get them.